core/domain/platform_parameter_list.py

```diff
@@ -39,6 +39,8 @@ class ParamNames(enum.Enum):
     ANDROID_BETA_LANDING_PAGE = 'android_beta_landing_page'
     BLOG_PAGES = 'blog_pages'
     DIAGNOSTIC_TEST = 'diagnostic_test'
+    SERIALIZE_CHAPTER_CURRICULUM_ADMIN_VIEW = (
```

Comment:
Please avoid starting with a verb for variables (see [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#javascript)).

Reply:
Changed.